bottlerocket/samples/RUNBOOK.md

```diff
@@ -69,6 +69,37 @@ EOF
 " 2> /dev/null
 ```
 
+### Workload Testing on `aws-ecs` Variants
+
+_Note_: An example of a workload test that can be used with the `ecs-workload-agent` is the `nvidia-smoke` test. See [README.md](../tests/workload/nvidia-smoke/README.md) for instructions on how to build and use its image.
```

Comment:
Not something I think needs to hold anything up, but if you do end up updating for some other reason I think it would be good to change the link from just being "README.md" to something like "the nvidia-smoke README".

Reply:
I realized that that might get confusing but wasn't sure if that warranted more specification. Since you brought it up, might as well change it. Thanks!